helm/charts/determined/values.yaml

```diff
@@ -17,7 +17,7 @@ masterPort: 8080
 # users will typically have to configure an Ingress to make the Determined master reachable from
 # outside the cluster. NodePort service is recommended when configuring TLS termination in a
 # load-balancer.
-useNodePortForMaster: false
```

Comment:
Let's remove this and make the service purely ClusterIP. Do that by removing these two rows:

https://github.com/coreweave/coreweave_determined/blob/rahul/roles/helm/charts/determined/templates/master-service.yaml#L14-L15